org.natron.Natron.json

```diff
@@ -0,0 +1,856 @@
+                
+                "--enable-gpl",
+                "--enable-libx264",
+                "--enable-libx265",
+                "--enable-libxvid",
+                "--enable-version3"
+            ],
+            "sources": [
+                {
+                    "type": "archive",
+                    "url": "https://www.ffmpeg.org/releases/ffmpeg-4.0.tar.xz",
+                    "sha256": "ed945daf40b124e77a685893cc025d086f638bc703183460aff49508edb3a43f"
+                }
+            ],
+            "cleanup": [
+                "/include",
+                "/lib/pkgconfig"
+            ]
+        },
+        {
+            "name": "sip",
+            "buildsystem": "simple",
+            "sources": [
+                {
+                    "type": "archive",
+                    "url": "https://sourceforge.net/projects/pyqt/files/sip/sip-4.19.8/sip-4.19.8.tar.gz",
+                    "sha256": "7eaf7a2ea7d4d38a56dd6d2506574464bddf7cf284c960801679942377c297bc"
+                }
+            ],
+            "build-commands": [
+                "python ./configure.py -b /app/bin -d /app/lib/python2.7/site-packages -e /app/include",
+                "make",
+                "make install"
+            ],
+            "cleanup": [
+                "/include"
+            ]
+        },
+        {
+            "name": "seexpr",
+            "buildsystem": "cmake",
+            "config-opts": [
+                "-DCMAKE_BUILD_TYPE=release",
+                "-DCMAKE_INSTALL_PREFIX=/app",
+                "-DCMAKE_INSTALL_LIBDIR=lib"
+            ],
+            "sources": [
+                {
+                    "type": "archive",
+                    "url": "https://github.com/wdas/SeExpr/archive/v1-2.11.tar.gz",
+                    "sha256": "b1395e1e87bdc75b6abc65ab8c4894b1e04ddb60c6eef277607bb6d44b31a6c2"
+                },
+                {
+                    "type": "patch",
+                    "path": "patches/seexpr-nodemos-notests.patch"
+                },
+                {
+                    "type": "patch",
+                    "path": "patches/seexpr-fix-inline-error.patch"
+                },
+                {
+                    "type": "patch",
+                    "path": "patches/seexpr-fix-cxxflags.patch"
+                }
+            ],
+            "cleanup": [
+                "src/SeExprEditor/generated/",
+                "src/SeExpr/generated/"
+            ]
+        },
+        {
+            "name": "natron",
+            "buildsystem": "simple",
+            "no-autogen": true,
+            "sources": [
+                {
+                    "type": "archive",
+                    "url": "https://github.com/NatronGitHub/Natron/archive/2.3.10.tar.gz",
+                    "sha256": "2aafe8376d653dbb5d835dcc9700b3b784ed840787df4d881a83c97cdacf9301"
+                },
+                {
+                    "type": "file",
+                    "path": "config.pri"
```

Comment:
Is this file upstreamed?

Reply:
This file needs to create manualy before build